seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -765,7 +765,7 @@
       "title": "Nest an Anchor Element within a Paragraph",
       "description": [
         "Again, here's a diagram of an <code>a</code> element for your reference:",
-        "<a href=\"//i.imgur.com/hviuZwe.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
+        "<a target=\"_blank\" href=\"//i.imgur.com/hviuZwe.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
```

Comment:
Good shout! I'll add that in!

What do you think about the `data-lightbox="img-enlarge"` attribute? Does it serve any purpose?

Do I do the other challenges as part of this PR? Or make a separate Issue for it?